src/sidebar/components/datalist.js

```diff
@@ -0,0 +1,105 @@
+import { createElement } from 'preact';
+import classnames from 'classnames';
+import propTypes from 'prop-types';
+import { useMemo } from 'preact/hooks';
+
+const defaultListFormatter = item => item;
+
+/**
+ * Custom datalist component. Use this in conjunction with an input field.
+ */
```

Comment:
Can you add a note here about the W3 accessibility design pattern that the datalist + input field are together intended to implement. This will provide guidance about the usage of eg. the `id` prop.